node_modules/gh-rest/lib/rest/user.js

```diff
@@ -33,6 +33,29 @@ module.exports = function(client) {
     };
 
     /**
+     * Get all users for an app
+     *
+     * @param  {Number}         [app]                           The application to search users in, defaults to the current application
```

Comment:
Should be `The id of the app to get the users for. Defaults to the current app` to be consistent with the other rest files.